### ISSUE TITLE
make the market_pred_XGBoost notebook achieve better accuracy

### ISSUE THREAD

**Issue by mijanr** [mijanr/PredEngine#3]
the current baseline XGBoost model performs okay. try adding new features, and probably do hyperparameter tuning for the xgboost model
